changelog/drnZ9nrASL2v6iCsnQdx2w.md

```diff
@@ -0,0 +1,4 @@
+audience: developers
+level: silent
+---
+Add a link to community-tc-config repo in EXPECTATIONS.md
```

Comment:
Changelogs are [phrased in the present tense, as if the change had just been deployed](https://github.com/taskcluster/taskcluster/blob/main/dev-docs/best-practices/changelog.md)
```suggestion
EXPECTATIONS.md now contains a link to the mozilla/community-tc-config repository.
```
(that's a little bit awkward for this change, but that's because it's a developer-doc change)